lib/components/primitive-components/Group/Group.ts

```diff
@@ -714,4 +718,47 @@ export class Group<Props extends z.ZodType<any, any, any> = typeof groupProps>
     const autorouter = this._getAutorouterConfig()
     return autorouter.groupMode === "sequential-trace"
   }
+
+  doInitialPcbDesignRuleChecks() {
+    if (this.root?.pcbDisabled) return
+    if (this.getInheritedProperty("routingDisabled")) return
+    const { db } = this.root!
+
+    if (this.isSubcircuit) {
+      const subcircuitComponentsByName = new Map<string, PrimitiveComponent[]>()
```

Comment:
Subcircuit component name checking is good 👍 this one doesnt need to go on the board

Reply:
So, we don't need to check it here in Board. But we will have to import from the Group class for this behaviour to be accepted, right?

Reply:
You can call the super method inside the board so that it runs the DRC checks that run for any subcircuit. You can alternatively create an internal underscore function that does what the subcircuit does. For example _checkForConflictingNames()